modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -1480,13 +1484,26 @@ void ONNXImporter::parseSplit(LayerParams& layerParams, const opencv_onnx::NodeP
         }
         layerParams.set("slice_point", DictValue::arrayInt(&slicePoints[0], slicePoints.size()));
     }
+    else if (node_proto.input_size() == 2) // opset >= 13, the split will be stored at the second input, instead of the attribute.
```

Comment:
BTW, there `onnx_opset` field to avoid guessing using the number of inputs (that is not reliable).

Reply:
Thx for your code review. I agree with you, but currently, I have no better solution for this.
Since we can not get the info of `opset` version.
And from onnx's doc, the split has only input when the `opset < 13`.
https://github.com/onnx/onnx/blob/main/docs/Changelog.md#split-11
And when `opset >= 13`, input may have the second input, and it's used only for `split`.
https://github.com/onnx/onnx/blob/main/docs/Operators.md#split.